subsys/zigbee/osif/zb_nrf_transceiver.c

```diff
@@ -325,11 +325,7 @@ zb_bool_t zb_trans_transmit(zb_uint8_t wait_type, zb_time_t tx_at,
 			return ZB_FALSE;
 		}
 
-		struct net_ptp_time timestamp = {
-			.second = tx_at / USEC_PER_SEC,
-			.nanosecond = (tx_at % USEC_PER_SEC) * NSEC_PER_USEC
-		};
-		net_pkt_set_timestamp(pkt, &timestamp);
+		net_pkt_set_txtime(pkt, (uint64_t)tx_at * NSEC_PER_USEC);
```

Comment:
if we decide to enable NET_PKT_TXTIME in samples, this line has to be put inside `#ifdef CONFIG_NET_PKT_TXTIME`